docs/UserGuide.md

```diff
@@ -360,6 +360,25 @@ Format: `delete_session n/STUDENT_NAME i/SESSION_INDEX`
 Example:
 * `delete_session n/John Lee i/1` deletes John Lee's **first** session
 
+#### Deleting a recurring tuition session: `delete_rec_session`
+
+Deletes a single session from an existing recurring tuition session of a specified student from TutorBuddy.
```

Comment:
Maybe including the description here about splitting up the remaining sessions into 2 (All the sessions that happens before the deleted session are in one recurring session, all sessions after the deleted session are created in another recurring session) might be better?

Reply:
I briefly included it on line 378. Do you think it is better if I brought it up to line 365?

Reply:
Okay I went ahead to add the extra description on line 364. Let me know if this works.

Reply:
Yeap! Looks better! Thanks thanks!